packages/react-testing/src/element.ts

```diff
@@ -193,26 +197,30 @@ export class Element<Props> implements Node<Props> {
   trigger<K extends FunctionKeys<Props>>(
     prop: K,
     ...args: DeepPartialArguments<Props[K]>
-  ): ReturnType<
-    NonNullable<
-      Props[K] extends ((...args: any[]) => any) | undefined ? Props[K] : never
-    >
-  > {
+  ): NonNullable<Props[K]> extends (...args: any[]) => any
+    ? ReturnType<NonNullable<Props[K]>>
+    : any {
```

Comment:
Yeah I should have pointed that out. I was getting some weird issues where ts was complaining about `trigger` even though it was never touched. Perhaps some kind of bug? Anyway changing it to `any` resolved that and doesn't change type safety as there's already a constraint to ensure that the string prop we pass in resolves to a function so it will never be resolved to `any` anyway. 

I tried changing it back to `never` just now and I no longer see the type error, so I have no idea what that was. I've reverted it back to `never` though.